resolver_conn_wrapper_test.go

```diff
@@ -98,7 +98,10 @@ func TestDialParseTargetUnknownScheme(t *testing.T) {
 	} {
 		dialStrCh := make(chan string, 1)
 		cc, err := Dial(test.targetStr, WithInsecure(), WithDialer(func(t string, _ time.Duration) (net.Conn, error) {
-			dialStrCh <- t
+			select {
+			case dialStrCh <- t:
```

Comment:
Please don't shadow "t" in a test; this confused me.

Reply:
Done :)